cronjob/values.yaml

```diff
@@ -0,0 +1,23 @@
+image:
```

Comment:
- `image.name`is not required (`image.repository`+ `image.tag` are used)
- can we use email-retry here as default values?